README.md

```diff
@@ -58,6 +58,12 @@ Released version
    jupyter-nbextension enable nglview --py --sys-prefix
 ```
 
+To use in Jupyter Lab you need to install appropriate extension:
+
+```bash
+jupyter labextension install  nglview-js-widgets
```

Comment:
I think `jupyter-labextension install @jupyter-widgets/jupyterlab-manager` is required too.

The reason jupyterlab-extension is not advertised here because it's evolving so quickly. 
I prefer to have exact working versions for now. Please see: https://github.com/arose/nglview/blob/master/devtools/nglview-jupyterlab.sh

Reply:
It's evolving quickly sure but at the moment this is the only way to install the nglview extension so better add it here don't you think? First time I tried to install nglview I had to dig in the issues to find the name of the JLab extension.